e2e_test/e2e_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+from threading import Thread
+import app
+import pytest
+from selenium  import webdriver
+from dotenv import load_dotenv, find_dotenv
+from data.trello_items import Trello_service
+import time
+from selenium.webdriver.common.by import By
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Firefox() as driver:
+        yield driver
+
+@pytest.fixture(scope="module")
+def test_app(driver):
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    # Create the new board & update the board id environment variable
+    service = Trello_service()
+    service.initiate()
```

Comment:
Again, I'm not sure this service is used, so we should remove this, unless we're going to start using it for the functionality to add/delete a board - something like:
```python
    service = Trello_service()
    service.initiate()
    board_id = service.create_board() # this function to be written
    list_ids = service.get_lists()
    os.environ['TRELLO_BOARD_ID'] = board_id
    set_list_environment_variables() # this to be written
```